### PR TITLE
`<PageHeader/>`- subtitle ellipsis for 2 lines

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Apply ellipsis when subtitle is more than 2 lines. 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5840/

File: src/PageHeader/PageHeader.st.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.root {
```
Comment (shlomitc) [inline src/PageHeader/PageHeader.st.css:1]: That’s very hacky don’t you think?

Comment (TalZa1810) [reply src/PageHeader/PageHeader.st.css:1]: I'm still investigating this :)

**Comment by TalZa1810** [comment]
I'll split this PR to a PR related to the `Ellipsis` change and a different one for `PageHeader`. Closing this one.

**Pull request closed without merging**